「プロフィール」ボタンを押下するとエラーになる場合がある

エラーになり、モーダルは表示されない模様。

該当と思われるCloudWatchのログ↓

```
The server responded with: {'ok': False, 'error': 'invalid_arguments', 'response_metadata': {'messages': ['[ERROR] failed to match all allowed schemas [json-pointer:/view]', '[ERROR] failed to match all allowed schemas [json-pointer:/view/blocks/0/text]', '[ERROR] must be more than 0 characters [json-pointer:/view/blocks/0/text/text]', '[ERROR] must be more than 0 characters [json-pointer:/view/blocks/0/text/text]']}})
```

--- a/bee_slack_app/view_controller/user.py
+++ b/bee_slack_app/view_controller/user.py
@@ -17,7 +17,10 @@ def user_controller(app):
         # slackアカウントから名前（Display Name）Display Nameを取得する
         # display_nameを設定していない場合は、設定必須のreal_nameをユーザ名とすることで、対応する
         user_info = client.users_info(user=user_id)
-        user_name = user_info["user"]["profile"]["display_name"] or user_info["user"]["profile"]["real_name"]
+        user_name = (
+            user_info["user"]["profile"]["display_name"]
+            or user_info["user"]["profile"]["real_name"]
+        )
 
         modal_view = generate_user_input_modal_view(user_name, user)
 

